.github/workflows/ci-cd.yml

```
@@ -0,0 +1,71 @@
+          script: |
+            const prNumber = context.payload.pull_request.number;
+            await github.rest.issues.createComment({
+              issue_number: prNumber,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: `🎩 빌드에 성공했습니다!`
+            });
+
+      - name: Comment on PR for build failure
+        if: ${{ github.event_name == 'pull_request' && failure() }}
+        uses: actions/github-script@v6
```

actionlint (1.7.7)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)